administrator/components/com_content/controllers/article.php

```diff
@@ -156,7 +156,9 @@ public function batch($model = null)
 	 */
 	protected function postSaveHook(JModelLegacy $model, $validData = array())
 	{
-
+		require_once JPATH_ADMINISTRATOR . '/components/com_menus/models/item.php';
```

Comment:
Instead of  `required_once`, 
```php
JModelLegacy::addIncludePath(JPATH_ADMINISTRATOR . '/components/com_menus/models');
```
is the valid way to include model path.